agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -1398,14 +1398,18 @@ public static class ProfilerConfiguration {
     public int configPollPeriodSeconds = 60;
     public int periodicRecordingDurationSeconds = 120;
     public int periodicRecordingIntervalSeconds = 60 * 60;
-    public boolean enabled = false;
+    @Nullable public Boolean enabled;
```

Comment:
does this work (and help simplify?)
```suggestion
    public boolean enabled = true;
```